llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7482,12 +7482,11 @@ VPRecipeBuilder::tryToWidenMemory(Instruction *I, ArrayRef<VPValue *> Operands,
     VPSingleDefRecipe *VectorPtr;
     if (Reverse) {
       // When folding the tail, we may compute an address that we don't in the
-      // original scalar loop and it may not be inbounds. Drop Inbounds in that
-      // case.
+      // original scalar loop: use none in this case.
```

Comment:
Nit, mention what's being dropped?
```suggestion
      // original scalar loop: drop nusw/inbounds in this case.
```